modules/nw-kuryr-migration.adoc

```diff
@@ -250,24 +250,6 @@ You cannot change the service network address block during the migration.
 You cannot use any CIDR block that overlaps with the `100.64.0.0/16` CIDR block because the OVN-Kubernetes network provider uses this block internally.
 ====
 
-. Verify that the Multus daemon set rollout is complete by entering the following command:
```

Comment:
Hey @gryf , the Multus verification still exists in the OpenShift-SDN to OVN-Kubernetes migration procedure. [0]
How does this kind of migration different? Should the Multus verification should be deleted from there as well? 

[0] https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-migration_migrate-from-openshift-sdn

Reply:
Yes, it's there, but it'll be changed AFAIK.